app/css/interceptor.css

```diff
@@ -743,8 +743,7 @@ svg.spinner > circle {
 	padding: 10px;
 	margin: auto;
 	margin-bottom: 10px;
-	min-width: 400px;
-	max-width: fit-content;
+	max-width: min(400px, 100%);
```

Comment:
I'm not a fan of fixed pixel count max width.  Why is this necessary?  What does it look like on a very small screen?  What about a very big screen?  What happens in both cases if you just set it to 100%?

Reply:
yeah, `fit-content` is actually what we want. It also fix the issue I had yesterday with this

Reply:
and get rid of the min-width that we used to have